plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -42,6 +41,8 @@ import (
 	"k8s.io/kubernetes/plugin/pkg/scheduler/schedulercache"
 	schedutil "k8s.io/kubernetes/plugin/pkg/scheduler/util"
 	"k8s.io/metrics/pkg/client/clientset_generated/clientset"
+
+	"github.com/golang/glog"
```

Comment:
This isn't really doing much and TBH the codebase is inconsistent on it's ordering.